duktape/src/main/jni/duktape-jni.cpp

```diff
@@ -79,33 +79,33 @@ Java_com_squareup_duktape_Duktape_evaluate__JLjava_lang_String_2Ljava_lang_Strin
 }
 
 JNIEXPORT void JNICALL
-Java_com_squareup_duktape_Duktape_bind__JLjava_lang_String_2Ljava_lang_Object_2_3Ljava_lang_Object_2(
-    JNIEnv *env, jclass type, jlong context, jstring name, jobject object, jobjectArray methods) {
+Java_com_squareup_duktape_Duktape_bind(JNIEnv *env, jclass type, jlong context, jstring name,
+                                       jobject object, jobjectArray methods) {
   DuktapeContext* duktape = reinterpret_cast<DuktapeContext*>(context);
   if (throwIfNull(env, duktape)) {
     return;
   }
   duktape->bind(env, name, object, methods);
 }
 
-JNIEXPORT jobject JNICALL
+JNIEXPORT jlong JNICALL
 Java_com_squareup_duktape_Duktape_proxy(JNIEnv *env, jclass type, jlong context, jstring name,
                                         jobjectArray methods) {
   DuktapeContext* duktape = reinterpret_cast<DuktapeContext*>(context);
   if (throwIfNull(env, duktape)) {
-    return nullptr;
+    return 0L;
```

Comment:
Sure, but we've already thrown a NullPointerException :-).   I'll change it to return -1L to make it clear.